zingolib/src/wallet/transaction_records_by_id.rs

```diff
@@ -585,9 +589,10 @@ mod tests {
             pending_spent: true,
             spent: true,
         };
-        let spentish_notes_in_tx_cvnwis = transaction_record_cvnwis.query_for_ids(
-            OutputQuery::new(query_for_spentish_notes, OutputPoolQuery::any()),
-        );
+        let spentish_notes_in_tx_cvnwis = transaction_record_cvnwis.query_for_ids(OutputQuery {
```

Comment:
spentish? cvnwis? im non blocking on this as already in the codebase but how is this *sludge* getting into the codebase?